### DISCUSSION TITLE
export shows N/A for income events for syslog:udp

### DISCUSSION THREAD

**Discussion by mhbitman** [vectordotdev/vector#21198] [General]
In my configuration there is one syslog source with type:UDP
in `vector top` and exporter I see N/A for input events while I can see their messages as output events for the same component.

Did I miss sth in configuration?
![Screenshot 2024-08-30 at 13 53 56](https://github.com/user-attachments/assets/ee715b70-1048-413f-b194-16eb01a6cd95)

**Comment by jszwedko** [comment]
Hmm, that seems like it might be a bug. Would you mind opening a bug report with the information requested by the [bug report template](https://github.com/vectordotdev/vector/issues/new?assignees=&labels=type%3A+bug&projects=&template=bug.yml)?

**Reply by mhbitman** [reply]
#21206
